array_methods_lab.js

```diff
@@ -0,0 +1,227 @@
+// # Exercises
+//
+// 1. Given an array of numbers, write a function that returns a new array where every element is incremented by 10.
+console.log("Number 1")
+let arr = [1, 2, 3, 4, 5]
+
+let addTen = arr.map((elem) => {
+  return elem + 10
+})
+
+console.log(addTen);
+
+// 2. Given an array, return a new array where every non-string element is converted to an empty string.
+console.log("Number 2")
+let arr2 = ["a", 123, "b", "c", {name: "cat"}]
+
+
+let onlyStrings = arr2.map((elem) => {
```

Comment:
This code wouldn't give us good answer since it will replace all non-string elements with an empty array (""). Better thing to do in this case is to do the following: 
arr2.map(elem => {
if(typeof elem === "string"){
return elem
}
})

Above code will return only strings into new array that is created by map method.